logstash/conf/loki.conf

```diff
@@ -0,0 +1,49 @@
+input {
+#  generator { message => "Hello world!" count => 1 }
+  beats {
+    port => 5044
+  }
+}
+
+output {
+  loki {
+    url => "http://localhost:3100"
+
+    # Array specifiying the extracted labels. This is a mandatory field. A minimum of 1 label should be provided. 
+    # All nested labels are extracted by prefixing the parent key i.e if there is a nested label like
+    # {log => {file => {path => "/path/to/file.log"}}}, extratced label would be "{log_file_path => /path/to/file.log}"
+    include_labels => ["agent_version", "@version", "host_name", "log_file_path"]
```

Comment:
Since this is the default files can we tweak to be clean. for instance.

include_labels => ["host_name", "log_file_path"]

should be enough as default.

Reply:
Nevermind the default file is elsewhere. What's this one for ?

Reply:
I removed it.